inc/custom_types/dates_type.php

```diff
@@ -0,0 +1,109 @@
+        'name_admin_bar'        => _x( 'Termin', 'Add New on Toolbar', 'lauch' ),
+        'add_new'               => __( 'Neues hinzufügen', 'lauch' ),
+        'add_new_item'          => __( 'Neuen Termin hinzufügen', 'lauch' ),
+        'new_item'              => __( 'Neuen Termin', 'lauch' ),
+        'edit_item'             => __( 'Termin bearbeiten', 'lauch' ),
+        'view_item'             => __( 'Termin ansehen', 'lauch' ),
+        'all_items'             => __( 'Alle Termine', 'lauch' ),
+        'search_items'          => __( 'Termine durchsuchen', 'lauch' ),
+        'parent_item_colon'     => __( 'Parent Termine:', 'lauch' ),
+        'not_found'             => __( 'No dates found.', 'lauch' ),
+        'not_found_in_trash'    => __( 'No dates found in Trash.', 'lauch' ),
+        'featured_image'        => _x( 'Termin-Bild', 'Overrides the “Featured Image” phrase for this post type. Added in 4.3', 'lauch' ),
+        'set_featured_image'    => _x( 'Termin-Bild bearbeiten', 'Overrides the “Set featured image” phrase for this post type. Added in 4.3', 'lauch' ),
+        'remove_featured_image' => _x( 'Termin-Bild entfernen', 'Overrides the “Remove featured image” phrase for this post type. Added in 4.3', 'lauch' ),
+        'use_featured_image'    => _x( 'Als Termin-Bild ', 'Overrides the “Use as featured image” phrase for this post type. Added in 4.3', 'lauch' ),
+        'archives'              => _x( 'Termin Archiv', 'The post type archive label used in nav menus. Default “Post Archives”. Added in 4.4', 'lauch' ),
+        'insert_into_item'      => _x( 'Insert into Termine', 'Overrides the “Insert into post”/”Insert into page” phrase (used when inserting media into a post). Added in 4.4', 'lauch' ),
+        'uploaded_to_this_item' => _x( 'Uploaded to this date', 'Overrides the “Uploaded to this post”/”Uploaded to this page” phrase (used when viewing media attached to a post). Added in 4.4', 'lauch' ),
+        'filter_items_list'     => _x( 'Filter dates list', 'Screen reader text for the filter links heading on the post type listing screen. Default “Filter posts list”/”Filter pages list”. Added in 4.4', 'lauch' ),
+        'items_list_navigation' => _x( 'Termine list navigation', 'Screen reader text for the pagination heading on the post type listing screen. Default “Posts list navigation”/”Pages list navigation”. Added in 4.4', 'lauch' ),
+        'items_list'            => _x( 'Termine list', 'Screen reader text for the items list heading on the post type listing screen. Default “Posts list”/”Pages list”. Added in 4.4', 'lauch' ),
+    );
+
+    $args = array(
+        'labels'             => $labels,
+        'public'             => true,
+        'publicly_queryable' => true,
+        'show_ui'            => true,
+        'show_in_menu'       => true,
+        'show_in_rest'       => true,
+        'query_var'          => true,
+        'capability_type'    => 'event',
```

Comment:
'event' wird schon in event_type.php benutzt und damit weise ich dann auch Schreibrechte zu. Wenn ich die Dok richtig verstehe, kann man hier eigene Types einsetzen, daher besser 'date' (hoffe das ist nirgends anders belegt)